src/NATS.Client.JetStream/Internal/NatsJSOpsDefaults.cs

```diff
@@ -10,13 +10,11 @@ internal static class NatsJSOpsDefaults
     private static readonly TimeSpan HeartbeatMin = TimeSpan.FromSeconds(.5);
 
     internal static (long MaxMsgs, long MaxBytes, long ThresholdMsgs, long ThresholdBytes) SetMax(
```

Comment:
I'm a little late in catching this, but I think this Class Name is supposed to be `NatsJSOptsDefaults`? (missing t in Opts)

Not a huge deal since it's internal